doc/admin-guide/files/records.config.en.rst

```diff
@@ -432,6 +432,14 @@ Thread Variables
    HTTP/2 graceful shutdown for now. Stopping Traffic Server here means sending
    `traffic_server` a signal either by `bin/trafficserver stop` or `kill`.
 
+.. ts:cv:: CONFIG proxy.config.thread.max_heartbeat_mseconds INT 60
+
+   Set the maximum heartbeat in milliseconds for threads, ranges from 0 to 1000.
+
+   This controls the maximum wait time of `epoll_wait`. On a system that is not busy,
```

Comment:
Don't say `epoll_wait` - rather something like "maximum amount of time the event loop will wait for I/O activity". Also add some verbiage about this being for expert system tuners, with random fiddling unlikely to be of benefit.